versions.mk

```diff
@@ -24,7 +24,12 @@ vVERSION := v$(VERSION:v%=%)
 GOLANG_VERSION ?= 1.20.4
 CUDA_VERSION ?= 11.8.0
 
+CLIENT_GEN_VERSION ?= v0.26.1
+CONTROLLER_GEN_VERSION ?= v0.9.2
+GOLANGCI_LINT_VERSION ?= v1.52.0
+MOQ_VERSION ?= v0.3.4
```

Comment:
Retracting, given my coment above.

Reply:
I would still remove them from here and only specify them in the Dockerfile. What motivation is there for keeping them defined here?

If these are "real" dependencies, then we should manage them using something else so that we can better manage the update lifecycle.